cc/mcts_node.cc

```diff
@@ -106,19 +106,20 @@ MctsNode::MctsNode(MctsNode* parent, Coord move)
 Coord MctsNode::GetMostVisitedMove() const {
   // Find the set of moves with the largest N.
   inline_vector<Coord, kNumMoves> moves;
-  moves.push_back(0);
-  int best_N = child_N(0);
-  for (int i = 1; i < kNumMoves; ++i) {
+  int best_N = -1;
+  for (int i = 0; i < kNumMoves; ++i) {
     int cn = child_N(i);
-    if (cn > best_N) {
-      moves.clear();
-      moves.push_back(i);
-      best_N = cn;
-    } else if (cn == best_N) {
+    if (cn >= best_N && children.contains(i)) {
```

Comment:
can the contains check be changed to && cn > 0?

Reply:
This is a weird edge case that I'm probably not dealing with in quite the best way. I've added an explanatory comment and a TODO.